### DISCUSSION TITLE
Source code for `org.lsposed.manager.databinding.*`

### DISCUSSION THREAD

**Discussion by infinity0** [LSPosed/LSPosed#2713] [Q&A] [answered]
Lots of java files from the app `import org.lsposed.manager.databinding.*` but this directory doesn't exist where it normally should.

How are these modules built and where is the source code for them?

**Comment by JasonKhew96** [comment, marked as answer]
https://developer.android.com/topic/libraries/data-binding/generated-binding

**Reply by infinity0** [reply]
Thanks!
